pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a group of members, teams, and applications with control over accessiblity and visiblity. Organizations allow granular access control at the application level through multiple roles that can be assigned to individual nembers or teams. This in turns helps to improve the visibility of who has access to which applications making collaboration easier. 
```

Comment:
* The words "accessiblity", "visiblity", "nembers" have typos in them - it would be a good idea to run a spell checker over the whole document.
* I'd rephrase _"with control over accessibility and visibility"_ (especially in the first sentence of this document) because "accessibility" generally refers to making computer systems accessible to people with disabilities or impairments to vision, hearing, etc, such as using larger fonts, high contrast, closed captions in video, etc. I think you meant access control instead of accessibility. _"control over visibility"_ leaves one wondering, _"visibility over what?"_ or _"why would I want to control visibility?"..._
* _"a group of members" (especially in the first sentence of this document) may sound puzzling to readers -- members of what, what members? -- not realizing that member means a balenaCloud user.

Suggestion:
```
An **organization** in balenaCloud is a group of users (members), teams, and applications,
with an access control system. Organizations allow the assignment of access control roles
to members or teams, improving visibility and reasoning over who has access to what applications
and, consequently, to device fleets.
```

By the way, in our quest for the soul of organizations, is an organization an access control system? Where "resources" are applications and "actors" are users and teams (which are groups or users). In which case:

```
An _organization_ in balenaCloud is an access control system for balena applications.
Organization _members_ are balenaCloud users who may be grouped in _teams,_ and
assigned _roles_ such as developer or operator. Organizations improve visibility and
reasoning over who has access to what applications and, consequently, to device fleets.

A balenaCloud account may contain one or more organizations, but a balena application
may be added to a single organization.
```

Reply:
My bad on the typos, I have checked the document before but will run the checker just to be safe. I liked both your suggestions so how about this as the final version. Pushed it just now.